devicon.json

```
@@ -8381,6 +8381,29 @@
             }
         ]
     },
+    {
+        "name": "reactrouter", 
+        "altnames": [
+            "reactrouter"
+        ],
+        "tags": [
+            "framework"
+        ], 
+        "versions": {
+            "SVG": [
+                "original",
+                "original-wordmark",
+                "plain",
+                "plain-wordmark"
+            ], 
+            "font": [
+                "original",
+                "original-wordmark"
```

The font versions should always (or in most cases) be the plain versions.
```suggestion
                "plain",
                "plain-wordmark"
```